luigi/local_target.py

```diff
@@ -71,7 +71,18 @@ def mkdir(self, path, parents=True, raise_if_exists=False):
                 return
 
         if parents:
-            os.makedirs(path)
+            # for Python 2 compatibility
+            try:
+                FileNotExistsError
+            except NameError:
+                FileNotExistsError = OSError
+
+            try:
+                os.makedirs(path)
+            except FileNotExistsError as err:
+                # somebody already created the path
```

Comment:
This comment is very helpful explaining why we can ignore the error. Thanks for adding it.